R/commonerrorcheck.R

```diff
@@ -767,3 +768,28 @@
 
   return(result)
 }
+
+.checkMissingRows <- function(dataset, target, grouping = NULL,
+                              maximumPercentageMissing = 30) {
+
+  result <- list(error = FALSE, errorVars = NULL)
+
+  # adapted from lavaan:::lav_data_missing_patterns
+  findMissingRows <- function(x) {
+    missingValues <- !is.na(x)
```

Comment:
Something is a missing value when it is *not* `NA`?

Reply:
yeah that doesn't really make sense :p